refinery/core/urls.py

```diff
@@ -76,5 +77,6 @@
     url(r'^analyses/(?P<uuid>' + UUID_RE + r')/$',
         AnalysesViewSet.as_view()),
     url(r'^openid_token/$',
-        OpenIDToken.as_view(), name="openid-token")
+        OpenIDToken.as_view(), name="openid-token"),
+    url(r'^sitestatistics/(?P<type>(deltas)|(totals)).csv', site_statistics)
```

Comment:
If site_statistics is no longer a DRF API view then should it be managed by DRF URL config?